files/fr/web/css/gradient/index.md

```diff
@@ -127,7 +127,7 @@ body {
 
 ### Dégradé conique
 
-Les couleurs de transitions s'appliquent progressivement autour d'un cercle. Un tel dégradé est généré avec la fonction {{cssref("conic-gradient","conic-gradient()")}}.
+Les couleurs de transitions s'appliquent progressivement autour d'un cercle. Un tel dégradé est généré avec la fonction {{cssxref("conic-gradient", "conic-gradient()")}}.
```

Comment:
We prefer links to macros for l10n-fr :)

```suggestion
Les couleurs de transitions s'appliquent progressivement autour d'un cercle. Un tel dégradé est généré avec la fonction [`conic-gradient()`](/fr/docs/Web/CSS/gradient/conic-gradient).
```

Reply:
What is the advantage of this approach?
I think that, on the contrary, it is better to use the corresponding macros (if they exist) instead of regular links.
I even made changes to mdn/content to replace regular links with macros to make the documents more consistent:
https://github.com/mdn/content/pull/31874 =)

Reply:
cf. https://github.com/mdn/translated-content/pull/18012#issuecomment-1918034865 :)